sources/operator.1.15.0.pxc-operator.json

```diff
@@ -0,0 +1,139 @@
+            "critical": false
+          },
+          "5.7.44-31.65": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.44-31.65",
+            "image_hash": "36fafdef46485839d4ff7c6dc73b4542b07031644c0152e911acb9734ff2be85",
+            "status": "recommended",
+            "critical": false
+          },
+          "5.7.42-31.65": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.42-31.65",
+            "image_hash": "9dab86780f86ec9caf8e1032a563c131904b75a37edeaec159a93f7d0c16c603",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.39-31.61": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.39-31.61",
+            "image_hash": "9013170a71559bbac92ba9c2e986db9bda3a8a9e39ee1ee350e0ee94488bb6d7",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.36-31.55": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.36-31.55",
+            "image_hash": "c7bad990fc7ca0fde89240e921052f49da08b67c7c6dc54239593d61710be504",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.34-31.51": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.34-31.51",
+            "image_hash": "f8d51d7932b9bb1a5a896c7ae440256230eb69b55798ff37397aabfd58b80ccb",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.42.0": {
+            "image_path": "percona/pmm-client:2.42.0",
+            "image_hash": "14cb96de47e3bc239bf285f22ec6f170b4a1181301b19100f5b7dc22c210bf8c",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.41.2": {
+            "image_path": "percona/pmm-client:2.41.2",
+            "image_hash": "16d2499c1cbcc1af51bd3752fe7623b0d0a319ee128b12d41cadf8080d1ce56b",
+            "status": "available",
+            "critical": false
+          },
+          "2.41.1": {
+            "image_path": "percona/pmm-client:2.41.1",
+            "image_hash": "b10b771da20150390c8151cd1a3213a43348ec699064c953b2ad10783f8d7b1c",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "proxysql": {
+          "2.6.3": {
+            "image_path": "percona/proxysql2:2.6.3",
+            "image_hash": "9e573dcec9961cd916c43a5a9dded7b3c03822f682171a22a7417dac7aeec521",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "haproxy": {
+          "2.8.5": {
+            "image_path": "percona/haproxy:2.8.5",
+            "image_hash": "fe63b0e7d503b25ab3d6f4aeb2e983868fe50bf85ebc91c713a990306718a8a2",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "backup": {
+          "8.0.35": {
+            "image_path": "perconalab/percona-xtradb-cluster-operator:1.15.0-pxc8.0-backup-pxb8.0.35",
```

Comment:
should we put full PXC version in image as well?

Reply:
we did not do it earlier and I do not want to have it only for one release (we have a plan to use official image starting from next release)